Plugins/Player/Player.cpp

```diff
@@ -1666,3 +1666,139 @@ NWNX_EXPORT ArgumentStack UpdateWind(ArgumentStack&& args)
 
     return {};
 }
+
+
+NWNX_EXPORT ArgumentStack UpdateSkyBox(ArgumentStack&& args)
+{
+    if (auto *pPlayer = Utils::PopPlayer(args))
+    {
+        if (auto pPlayerCreature = Utils::AsNWSCreature(pPlayer->GetGameObject()))
+        {
+            
+            if  (auto* pArea = pPlayerCreature->GetArea())
+            {
+                if (pArea != NULL)
```

Comment:
The above `if (auto* area =...)` check implies this, so no need to repeat it. Same in other functions